packages/components/socialFeed/NewsFeed/NewsFeedInput.tsx

```diff
@@ -148,31 +147,10 @@ export const NewsFeedInput = React.forwardRef<
     const [isUploadLoading, setIsUploadLoading] = useState(false);
     const [isProgressBarShown, setIsProgressBarShown] = useState(false);
     const [premium, setPremium] = useState(false);
-    const [isShowMap, setShowMap] = useState(false);
-    const [locationSelected, setLocationSelected] = useState<LatLng>([
-      48.8566, 2.3522,
-    ]);
-    const [description, setDescription] = useState("");
-    const [developerMode] = useDeveloperMode();
-
-    console.log("locationSelectedlocationSelected", locationSelected);
-
-    // useEffect(() => {
-    //   if (Platform.OS === "web") {
-    //     navigator.geolocation.getCurrentPosition(
-    //       (position) => {
-    //         setLocationSelected([
-    //           position.coords.latitude,
-    //           position.coords.longitude,
-    //         ]);
-    //       },
-    //       (error) => {
-    //         console.error("Error getting geolocation:", error);
-    //       },
-    //     );
-    //   }
-    // }, []);
-
+    const [isMapShown, setIsMapShown] = useState(false);
+    const [location, setLocation] = useState<
+      CustomLatLngExpression | undefined
```

Comment:
no need to precis type undefined if default is already undefined

Reply:
Yes, I don't know why I did that